ethcore/src/engines/mod.rs

```diff
@@ -396,6 +393,11 @@ pub trait EthEngine: Engine<::machine::EthereumMachine> {
 	fn supports_wasm(&self) -> bool {
 		self.machine().supports_wasm()
 	}
+
+	/// Additional information.
+	fn additional_params(&self) -> HashMap<String, String> {
```

Comment:
Could probably be `BTreeMap` as well.

Reply:
agreed, but it's beyond the scope of this PR.
TBH, it's only used for the registry, so I don't see that much of a reason to make it a stringly-typed map in the first place.